samples/SwiftDemoApp/Podfile

```
@@ -1,8 +1,8 @@
 source 'https://cdn.cocoapods.org/'
-platform :ios, '13.0'
+platform :ios, '14.0'
 
 target 'SwiftDemoApp' do
   use_frameworks!
-  pod 'GoogleMaps', '~> 7.3'
+  pod 'GoogleMaps', '7.4.0'
```

```suggestion
  pod 'GoogleMaps', '8.0.0'
```
Shouldn't this be 8.0.0?

---

Yes, updated to latest